site/docs/foundations/data-visualization/color-and-pattern.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Color and pattern
+layout: DetailTechnical
+---
+
+Color is a valuable tool that helps effectively present data in your visualization. Using color correctly guides users’ attention and supports their understanding and interpretation of data.
+
+## Color usage
+
+Consider the purpose and impact of color in your chart. Color can be used to:
+
+**Show relationships**
+Use similar colors to imply that items are related. Contrasting colors, however, show that items are from distinct categories.
+
+**Represent data values**
+Color helps to give a clearer indication of range and how data values are distributed; for instance, highs and lows.
+
+**Highlight key elements**
+Use color to direct your audience's attention towards the main element or emphasize data that supports the story you’re communicating.
+
+**Indicate meaning**
+Some colors are strongly associated with certain objects, concepts or emotions. Leveraging these associations helps you communicate more efficiently and improves comprehension of your chart.
+
+<Image
+  src="/img/foundations/dataviz-color-usage.png"
+  srcDark="/img/foundations/dataviz-color-usage-dark.png"
+  alt="Diagram of bar chart in a hover state."
+/>
+
+## Categorical palette
+
+The categorical palette is designed to visualize distinct groups of items. It features contrasting colors with different hues, indicating the lack of visual relationship between the data sets. The categorical palette achieves a minimum of 3:1 contrast against each of the 3 supported background colors in Salt - in both light and dark mode.
+
+For more information, please refer to the [Categorical color page](/salt/foundations/color/categorical-colors).
+
+<Image
+  src="/img/foundations/dataviz-color-order.png"
+  srcDark="/img/foundations/dataviz-color-order-dark.png"
+  alt="Diagram of the categorical color palette."
+/>
+
+## Patterns
+
+According to [WCAG](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html), color shouldn’t be the only distinguishing factor to convey information about your data. To improve your chart’s accessibility for users with accessibility requirements, use additional visual cues, such as line and background patterns. The order of line patterns and pattern fills is pre-set.
+
+If you’re migrating from UITK to Salt, the line patterns and pattern fills used in the Salt charts are direct 1:1 copies from UITK’s patterns.
```

Comment:
I don't think this is necessary. I believe, as a rule, we don't mention UITK (other than the migratory theme) within the external website. We could say the same about the category colors etc, but we choose not to.